src/modules/user/coupon/CouponDrawerContent.tsx

```diff
@@ -0,0 +1,99 @@
+import { H3, H5, Text } from "@/modules/common/components/Typography";
+import { commaFormat } from "@/modules/common/utils";
+import { type Coupon } from "@/modules/user/mock/coupons";
+import styled from "@emotion/styled";
+import { Button } from "antd";
+import Image from "next/image";
+
+type CouponDrawerContentProps = {
+  coupon: Coupon | undefined | null;
```

Comment:
Can we kust use `coupon: Coupon | undefined;`

Reply:
Thx. Done.